code/modules/jobs/job_types/coroner.dm

```diff
@@ -0,0 +1,63 @@
+/datum/job/coroner
+	title = JOB_CORONER
+	description = "Perform Autopsies whenever needed, \
+		Update medical records accordingly, apply formaldehyde."
+	department_head = list(JOB_CHIEF_MEDICAL_OFFICER)
+	faction = FACTION_STATION
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the Head of Personnel and the Chief Medical Officer"
+	exp_granted_type = EXP_TYPE_CREW
+	config_tag = "CORONER"
+
+	outfit = /datum/outfit/job/coroner
+	plasmaman_outfit = /datum/outfit/plasmaman/medical
+
+	paycheck = PAYCHECK_CREW
+	paycheck_department = ACCOUNT_MED
+
+	display_order = JOB_DISPLAY_ORDER_CORONER
+	bounty_types = CIV_JOB_MED
+	department_for_prefs = /datum/job_department/medical
+	departments_list = list(
+		/datum/job_department/medical,
+		/datum/job_department/service,
+		)
```

Comment:
```suggestion
	)
```

Reply:
This is consistent with all other jobs, I'm not changing it

Reply:
new code should be better than old code bro

Reply:
new code it's been a year

Reply:
ok it's been 2 but it still stands

Reply:
the suggestion san made is correct and conformant to multi-lining style guidelines added within the last year. please adjust